Source/FullScreenPokemon.ts

```diff
@@ -3832,23 +3832,65 @@ module FullScreenPokemon {
 
         /**
          * Opens the context menu within the Pokedex menu for the selected Pokemon.
+         *
+         * @param settings The selected Pokemon.
          */
         openPokemonMenuContext(settings: any): void {
+            var moves: BattleMovr.IMove[] = settings.pokemon.moves,
+                options: any[],
+                i: number;
+
+            for (i = 0; i < moves.length; i += 1) {
+                if (moves[i].title = "Cut") {
```

Comment:
*If* you were to do it this way (which isn't happening), it would be better to do something like an `{ [i: string]: boolean }` listing which types are good, or `{ [i: string]: Function }` to map callbacks.